### PR TITLE
Make code compatible with Python 2 and 3

### PR DESCRIPTION
**Pull request by gcbeltramini**

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import time
```
Comment (hcarvalhoalves) [inline tests/test_core.py:2]: `from __future__ import print_function` so the test works both on 2.x and 3.

File: setup.py [hunk 2]
```diff
@@ -10,10 +10,12 @@
     url="https://github.com/hcarvalhoalves/python-pmap",
     package_dir={'': 'src'},
     packages=find_packages('src'),
-    version="1.2.0",
+    version="1.3.0",
     install_requires=[
```
Comment (hcarvalhoalves) [inline setup.py]: Now that it works on both versions, add the appropriate classifiers:

```
classifiers=[
"Programming Language :: Python :: 2.7",
"Programming Language :: Python :: 3.6"
]
```

File: setup.py [hunk 3]
```diff
@@ -10,10 +10,12 @@
     url="https://github.com/hcarvalhoalves/python-pmap",
     package_dir={'': 'src'},
     packages=find_packages('src'),
-    version="1.2.0",
+    version="1.3.0",
```
Comment (hcarvalhoalves) [inline setup.py]: Major bump?

Comment (gcbeltramini) [reply setup.py]: Since it's not a breaking change, I did a minor bump.

**Pull request merged**